CHANGES/687.feature

```diff
@@ -0,0 +1,7 @@
+Added the below filters to `pulp rpm content list`:
+- --arch-contains
+- --arch-startswith
+- --name-contains
+- --name-startswith
+- --release-contains
+- --release-startswith
```

Comment:
```suggestion
- `--arch-contains`
- `--arch-startswith`
- `--name-contains`
- `--name-startswith`
- `--release-contains`
- `--release-startswith`
```
I'm mildly concerned that between markdown and towncrier, this will bust the formatting. But we can make up for that later if you want.

Reply:
Indented-sublist has worked in the past - let's see if this change works for us.